dynatrace/src/test/java/com/dynatrace/opentelemetry/metric/CumulativeToDeltaConverterTest.java

```diff
@@ -212,4 +212,30 @@ void testDoubleAndLongDoNotInterfere() {
     assertThat(converter.convertDoubleTotalToDelta("test", createDoublePointData(200.5)))
         .isCloseTo(100.3, offset);
   }
+
+  @Test
+  void testAttributesAreDeDuplicated() {
+    // a delta is correctly calculated for the second metric, meaning that these two metrics are
+    // considered equal by the delta calculation. The duplicated attribute `attr1 is removed.
+    assertThat(
+            converter.convertLongTotalToDelta(
+                "test",
+                createLongPointData(
+                    100L, 0, Attributes.of(stringKey("attr1"), "v1", stringKey("attr2"), "v2"))))
+        .isNull();
+    assertThat(
+            converter.convertLongTotalToDelta(
+                "test",
+                createLongPointData( // has 'attr1' twice
```

Comment:
If we use two different attribute values we could not assert here since the behavior is then undefined and we have no idea what would happen, right?

Reply:
We do have an idea what happens. The `attributes` from OTel does the de-duplication and the last values wins. So if we have `attr1=v1` and `attr1=v2`, the final attributes list will contain `attr1=v2`. 

This will "break" the metric identification if it was reported first without `v2` and later reported with the duplicated attribute. Our code will think it's a new metric.

Reply:
Okay, so that means we could use `attr1=something random` and then `attr1=v1` here in the second metric point in order to make sure that the attributes are deduplicated based on their keys alone, right?

Reply:
Yes, correct. I added it now + another test to only make sure the OTel SDK does the correct sorting and deduplication.